meinberlin/assets/scss/components/_button.scss

```diff
@@ -16,7 +17,7 @@
 }
 
 // to be more specific then BS
```

Comment:
stands for Bootstrap which we have partially adopted, currently using BO for Berlin Online but if that's confusing then we can change it up, also I think that comment isn't needed anymore as I removed the `.btn` class as it will mess with future a4 updates so will remove comment